apps/src/templates/policy_compliance/AgeGatedStudentsModal/AgeGatedStudentsModal.tsx

```diff
@@ -0,0 +1,90 @@
+import React, {useEffect} from 'react';
+import {connect, useSelector} from 'react-redux';
+
+import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
+import {EVENTS, PLATFORMS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+import {RootState} from '@cdo/apps/types/redux';
+import i18n from '@cdo/locale';
+
+import BaseDialog from '../../BaseDialog';
+
+import AgeGatedStudentsTable from './AgeGatedStudentsTable';
+
+import styles from '@cdo/apps/templates/policy_compliance/AgeGatedStudentsModal/age-gated-students-modal.module.scss';
+
+interface ReduxState {
+  manageStudents: {
+    isLoadingStudents?: boolean;
+  };
+}
+interface Props {
+  onClose: () => void;
+  isOpen: boolean;
+  isLoadingStudents: boolean;
+  ageGatedStudentsCount?: number;
+}
+
+const AgeGatedStudentsModal: React.FC<Props> = ({
+  isLoadingStudents,
+  isOpen,
+  onClose,
+  ageGatedStudentsCount = 0,
+}) => {
+  const currentUser = useSelector((state: RootState) => state.currentUser);
+  const reportEvent = (eventName: string, payload: object = {}) => {
+    analyticsReporter.sendEvent(eventName, payload, PLATFORMS.AMPLITUDE);
+  };
+
+  const modalDocumentationClicked = () => {
+    reportEvent(EVENTS.CAP_STUDENT_WARNING_LINK_CLICKED, {
+      user_id: currentUser.userId,
+      number_of_gateable_students: ageGatedStudentsCount,
+    });
+  };
+
+  useEffect(() => {
+    reportEvent(EVENTS.CAP_AGE_GATED_MODAL_SHOWN, {
+      user_id: currentUser.userId,
+      number_of_gateable_students: ageGatedStudentsCount,
+    });
+  }, [currentUser.userId, ageGatedStudentsCount]);
+  return (
+    <BaseDialog
+      isOpen={isOpen}
+      handleClose={onClose}
+      useUpdatedStyles={true}
+      fixedWidth={800}
+    >
+      <div
+        className={styles.modalContainer}
+        data-testid="age-gated-students-modal"
+      >
+        <div>
+          <h2 className={styles.modalHeader}>
+            {i18n.childAccountPolicy_studentParentalConsentStatus()}
+          </h2>
+          <hr />
+          <p>{i18n.childAccountPolicy_studentParentalConsentNotice()}</p>
+          <br />
+          <a href="#" onClick={modalDocumentationClicked}>
```

Comment:
I created a [JIRA](https://codedotorg.atlassian.net/browse/P20-962) to figure out the actual value for this.

Reply:
Thanks! Logic is being add in the banner PR as well: https://github.com/code-dot-org/code-dot-org/pull/58481